server/datastore/mysql/hosts.go

```diff
@@ -3657,6 +3659,34 @@ func (ds *Datastore) SetOrUpdateHostEmailsFromMdmIdpAccounts(
 	)
 }
 
+func (ds *Datastore) SetOrUpdateHostEmailsFromMdmIdpAccountsByHostUUID(
```

Comment:
Nit, but I think the similar methods use `MDM` and `IdP` casing:
```suggestion
func (ds *Datastore) SetOrUpdateHostEmailsFromMDMIdPAccountsByHostUUID(
```